firefox Bug 1732325 

mmap in SharedMemory::FindFreeAddressSpace should use MAP_NORESERVE

diff --git a/ipc/chromium/src/base/shared_memory_posix.cc b/ipc/chromium/src/base/shared_memory_posix.cc
index 896cb98..1e0460a 100644
--- a/ipc/chromium/src/base/shared_memory_posix.cc
+++ b/ipc/chromium/src/base/shared_memory_posix.cc
@@ -513,7 +513,7 @@ bool SharedMemory::Map(size_t bytes, void* fixed_address) {
 
 void* SharedMemory::FindFreeAddressSpace(size_t size) {
   void* memory =
-      mmap(NULL, size, PROT_NONE, MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
+      mmap(NULL, size, PROT_NONE, MAP_PRIVATE | MAP_ANONYMOUS | MAP_NORESERVE, -1, 0);
   munmap(memory, size);
   return memory != MAP_FAILED ? memory : NULL;
 }
